src/index.js

```diff
@@ -54,115 +56,119 @@ const App = () => {
+                  ]}
+                />
+
+                <Main>
+
+                  <WrappedRequest url={data.instructor_url}>
+                    {({data}) => {
+                      const instructor = data
+                      return (
+                        <Switch>
+
+                          <Route 
+                            exact
+                            path='/'
+                            render={() => (
+                              <Dashboard instructor={instructor} />
+                            )}
+                          />
+
+                          <Route 
+                            exact
+                            path='/lessons/new'
+                            render={() => (
+                              <New instructor={instructor} />
+                            )}
+                          />
+
+                          <Route 
+                            path={`/lessons/:slug`}
+                            render={({match}) => (
+                              <WrappedRequest
+                                url={`/api/v1/lessons/${match.params.slug}`}
+                                subscribe
+                              >
+                                {({request, data}) => (
+                                  <Lesson 
+                                    instructor={instructor}
+                                    lesson={data} 
+                                    requestLesson={request}
+                                  />
+                                )}
+                              </WrappedRequest>
+                            )}
+                          />
+
+                          <Route 
+                            path='/lessons'
+                            render={() => (
+                              <Lessons instructor={instructor} />
+                            )}
+                          />
+
+                          <Route 
+                            path={`/instructors/:slug`}
+                            render={({match}) => (
+                              <WrappedRequest url={`/api/v1/instructors/${match.params.slug}`}>
+                                {({data}) => (
+                                  <Instructor instructor={data} />
+                                )}
+                              </WrappedRequest>
+                            )}
+                          />
+
+                          <Route
+                            path='/instructors'
+                            render={() => (
+                              <Instructors instructors={data} />
+                            )}
+                          />
+
+                          <Route component={RouteNotFound} />
+
+                        </Switch>
+
+                      )
+                    }}
+                  </WrappedRequest>
+                </Main>
+
+              </div>
+
+            </Localization>
           </BrowserRouter>
         )
       }}
```

Comment:
Could simplify this to a single return in the future so only one `Localization` wrapper is needed